src/lambda/events/generate_test_event.py

```diff
@@ -42,81 +45,111 @@
     "UnsubscribeURL": "https://sns.us-east-1.amazonaws.com/?Action=Unsubscribe&SubscriptionArn=arn:aws:sns:us-east-1:563295687221:phil_s3_bucket_update:89b889dd-9e68-41a5-b865-c194f09d024c"
 }
 
-def create_message_template(project_id, folder_id):
+def create_message_template(entity):
   return {
     "appId": "example-app-1",
-    "recordId": "-4I2GOqDSdjaXsbuw8oYXBKK",
+    "recordId": entity.annotations.recordId.pop(),
```

Comment:
Are there multiple record ids? or why are you doing a pop() here?

Reply:
For some reason every annotation is a list of one item. I don't know if Bridge is doing something peculiar to make the annotations formatted that way or something else is happening.

Reply:
Ya.. i'm not sure why this is the case either, but a list is what is returned. I was just curious if there was going to ever be more than one recordId.

Reply:
No, that should be impossible.